recipes/wiringpi/all/conanfile.py

```diff
@@ -32,16 +38,24 @@ def configure(self):
         self.settings.rm_safe("compiler.libcxx")
         self.settings.rm_safe("compiler.cppstd")
 
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
     def validate(self):
         if self.settings.os != "Linux":
             raise ConanInvalidConfiguration(f"{self.ref} only works for Linux")
-
-    def layout(self):
-        cmake_layout(self, src_folder="src")
+        if Version(self.version) >= 3.0:
+            if self.settings.compiler == "gcc" and \
+                Version(self.settings.compiler.version) < 8:
```

Comment:
Why does it need GCC 8 ?

Reply:
@uilianries 
using const variables are not valid in these versions.

```
src/wiringPi/wiringPi.c:144:39: error: initializer element is not constant
 const unsigned int RP1_INV_PUD_MASK = ~(RP1_PUD_UP | RP1_PUD_DOWN); //~0x0C
                                       ^
```

Reply:
@toge Thank you for pointing! 😄